app/models/agents/rss_agent.rb

```diff
@@ -139,21 +162,241 @@ def check_and_track(entry_id)
       end
     end
 
+    LINK_ATTRS = %i[href rel type hreflang title length]
+    ENCLOSURE_ATTRS = %i[url type length]
+
+    unless dependencies_missing?
+      class AtomAuthor
```

Comment:
Maybe.  Would `app/models/agents/rss_agent/feedjira_extension.rb` be a good place?